src/Ilios/CliBundle/Command/UpdateServiceWorkerCommand.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Ilios\CliBundle\Command;
+
+use Ilios\CoreBundle\Service\Fetch;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
+
+use Ilios\CoreBundle\Service\Filesystem;
+
+/**
+ * Pull down service worker JS files from cloudfront
+ * So they can be served from the API host since service workers
+ * won't work from a different domain
+ *
+ * Class UpdateServiceWorkerCommand
+ * @package Ilios\CliBUndle\Command
+ */
+class UpdateServiceWorkerCommand extends Command implements CacheWarmerInterface
+{
+    /**
+     * @var string
+     */
+    const SWJS_CACHE_FILE_NAME = '/ilios/sw.js';
+    const SW_REGISTRATION_CACHE_FILE_NAME = '/ilios/sw-registration.js';
+    const PRODUCTION = 'prod';
+    const STAGING = 'stage';
+    const DEVELOPMENT = 'dev';
+    const CDN_ASSET_DOMAIN = 'https://d26vzvixg52o0d.cloudfront.net/';
```

Comment:
I'm not sure what the alternative would be. We could put it in the configuration, but it would still be hardwired there as it isn't configurable by the user. It should probably be combined with the domain for the index file at some point, maybe refs #1862